whois-client/src/main/java/net/ripe/db/whois/query/QueryMessages.java

```diff
@@ -62,6 +60,13 @@ public static Message unvalidatedAbuseCShown(final CharSequence key, final CharS
                 "\nAbuse-mailbox validation failed. Please refer to %s for further information.", key, value, orgId);
     }
 
+    public static Message roaRouteConflicts(final long asn){
```

Comment:
Can you specify the object type in the text? Use either "route:" or "route6:" depending on the object type. Or, use the term "route(6)" to refer to either a route or route6 object.

Reply:
Done